tests/fastifyv2.test.js

```diff
@@ -0,0 +1,334 @@
+/* global describe, test, expect */
+'use strict'
+
+const Fastify = require('fastifyv2')
+const request = require('supertest')
+
+const rTracer = require('../index')
+
+const pluginType = 'plugin'
+const middlewareType = 'middleware'
+const types = [pluginType, middlewareType]
+
+const register = (type, app, options) => {
+  switch (type) {
+    case pluginType:
+      return app.register(rTracer.fastifyPlugin, options)
+    case middlewareType:
+      return app.use(rTracer.fastifyMiddleware(options))
+    default:
+      throw new Error('Unexpected type')
+  }
+}
+
+for (const type of types) {
+  describe(`cls-rtracer for Fastify@2 - ${type}`, () => {
```

Comment:
nit: make test name more consistent with koa v1 one
```suggestion
  describe(`cls-rtracer for Fastify v2 - ${type}`, () => {
```